src/js/common/helpers/login-helpers.js

```diff
@@ -4,28 +4,13 @@ import environment from './environment.js';
 import { updateLoggedInStatus } from '../../login/actions';
 import { updateProfileField } from '../../user-profile/actions';
 
-export function handleLogin() {
-  this.serverRequest = fetch(`${environment.API_URL}/v0/sessions/new?level=1`, {
-    method: 'GET',
-  }).then(response => {
-    return response.json();
-  }).then(json => {
-    const myVerifyUrl = json.authenticate_via_get;
-    const receiver = window.open(myVerifyUrl, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
-    receiver.focus();
-  });
-}
-
-export function handleVerify() {
-  this.serverRequest = fetch(`${environment.API_URL}/v0/sessions/new?level=3`, {
-    method: 'GET',
-  }).then(response => {
-    return response.json();
-  }).then(json => {
-    const myVerifyUrl = json.authenticate_via_get;
-    const receiver = window.open(myVerifyUrl, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
+export function handleVerify(verifyUrl) {
+  window.dataLayer.push({ event: 'verify-link-clicked' });
+  if (verifyUrl) {
```

Comment:
If `verifyUrl` is passed as `null`, this will be a no-op.  Can we instead have an `else` condition that fetches the verify URL (a.k.a. LOA3 url) then opens it in a new window for the unlikely event that `verifyUrl` is `null`?